charts/opensearch/templates/role.yaml

```diff
@@ -4,7 +4,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: {{ $fullName | quote }}
-    {{- include "opensearch.labels" . | nindent 4 }}
+  labels:
+  {{- include "opensearch.labels" . | nindent 4 }}
```

Comment:
I think the indent here should match the other templates.

Reply:
I don't mind retesting with the new indent. Let me know.

Reply:
@smlx Done.